tensorflow/java/src/main/java/org/tensorflow/SavedModelBundle.java

```diff
@@ -64,29 +78,25 @@ public void close() {
     private final Session session;
     private final byte[] metaGraphDef;
 
-    /**
-     * Load a saved model from an export directory.
-     * @param exportDir the directory path containing a saved model.
-     * @param tags the tags identifying the specific metagraphdef to load.
-     * @return a bundle containing the graph and associated session.
-     */
-    public static SavedModelBundle loadSavedModel(String exportDir, Set<String> tags) {
-        return load(exportDir, tags.toArray(new String[tags.size()]));
+    SavedModelBundle(Graph graph, Session session, byte[] metaGraphDef) {
+        this.graph = graph;
+        this.session = session;
+        this.metaGraphDef = metaGraphDef;
     }
 
     /**
-     * Create a SavedModelBundle object from a handle to the C TF_Graph object and
-     * to the C TF_Session object, plus the associated metagraphdef.
+     * Create a SavedModelBundle object from a handle to the C <i>TF_Graph</i> object and
```

Comment:
Use {@code ... } instead of <i>...</i>.

Or better still, since this is a private method and thus won't appear in Javadocs can do away with any annotations, which will make it easier to read in code.

.e.g, "handle to the C TF_Graph object and ..."